src/lib/db.ts

```diff
@@ -64,7 +81,7 @@ export class DB {
 
       this.unregisterPresence = () => {
         userPresence.remove();
-      }
+      };
```

Comment:
Interesting, it turns out tslint was silently failing for me because I hadn't run `npm install` and it was missing tslint-react. There's no indication of this, except errors aren't spotted. Thanks for fixing.